Take short answer

shengjiayuu opened the pull request:
1. update the question_log and score_log. All tests pass locally, but please double check.
2. update the changelog file up to this milestone.
3. updated tutorials for short-answer.
4. for the contributing file, i think it nothing need to add. But please let me know if i am wrong.Thanks!
<img width="568" alt="Screen Shot 2025-01-26 at 1 49 47 AM" src="https://github.com/user-attachments/assets/5224def5-6376-4a3f-9667-249c26edd334" />

shengjiayuu committed:
Update changelog for 3 releases

shengjiayuu committed:
Update the documentation and the save score and question functions

shengjiayuu committed:
add tutorial for the short-answer questions

cs-uche commented:
> 1. update the question_log and score_log. All tests pass locally, but please double check.
> 2. update the changelog file up to this milestone.
> 3. updated tutorials for short-answer.
> 4. for the contributing file, i think it nothing need to add. But please let me know if i am wrong.Thanks!
> 
> <img alt="Screen Shot 2025-01-26 at 1 49 47 AM" width="568" src="https://private-user-images.githubusercontent.com/58960224/406727346-5224def5-6376-4a3f-9667-249c26edd334.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc4OTM4NjcsIm5iZiI6MTczNzg5MzU2NywicGF0aCI6Ii81ODk2MDIyNC80MDY3MjczNDYtNTIyNGRlZjUtNjM3Ni00YTNmLTk2NjctMjQ5YzI2ZWRkMzM0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTI2VDEyMTI0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQxYzI1NmZhYTVhYzRiODg4OTk4NDBjZDQwZDdiNTUyMWIwMTQ2ZTQ0YTYyOWJlNGNlYzVhMTVjYzdjMmE4MjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.mlajBiOIHMnAl5U-wCcPAiVuZppKbJp7Ha237x7EhhM">

@shengjiayuu For the contributing file please view the [backlog](https://github.com/UBC-MDS/dsci524_group13_quizit/issues/24#issue-2803120668) include the conda create and activate steps and other steps a developer would need to edit the package

shengjiayuu committed:
Update CONTRIBUTING.md

shengjiayuu commented:
> > 1. update the question_log and score_log. All tests pass locally, but please double check.
> > 2. update the changelog file up to this milestone.
> > 3. updated tutorials for short-answer.
> > 4. for the contributing file, i think it nothing need to add. But please let me know if i am wrong.Thanks!
> > 
> > <img alt="Screen Shot 2025-01-26 at 1 49 47 AM" width="568" src="https://private-user-images.githubusercontent.com/58960224/406727346-5224def5-6376-4a3f-9667-249c26edd334.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc4OTM4NjcsIm5iZiI6MTczNzg5MzU2NywicGF0aCI6Ii81ODk2MDIyNC80MDY3MjczNDYtNTIyNGRlZjUtNjM3Ni00YTNmLTk2NjctMjQ5YzI2ZWRkMzM0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTI2VDEyMTI0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQxYzI1NmZhYTVhYzRiODg4OTk4NDBjZDQwZDdiNTUyMWIwMTQ2ZTQ0YTYyOWJlNGNlYzVhMTVjYzdjMmE4MjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.mlajBiOIHMnAl5U-wCcPAiVuZppKbJp7Ha237x7EhhM">
> 
> @shengjiayuu For the contributing file please view the [backlog](https://github.com/UBC-MDS/dsci524_group13_quizit/issues/24#issue-2803120668) include the conda create and activate steps and other steps a developer would need to edit the package

Thanks so much for reminding me! added more steps into the contributing file.

MavisWong295 commented:
Don't see any issues here
